### PR TITLE
Feature OSM tutorial

### PR DESCRIPTION
**Pull request by DavidMansolino**
The goal of this PR is to provide a step by step tutorial explaining how to import a map from OSM in Webots and the interface it with SUMO (there is more and more user question about this).

  - [x] Add note about 3D and sumo not compatible

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: automobile/menu.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
 - [Tools](tools.md)
     - [OpenStreetMap importer](openstreetmap-importer.md)
     - [SUMO interface](sumo-interface.md)
+    - [Real world scenario creation tutorial](scenario-creation-tutorial.md)
```
Comment (omichel) [inline automobile/menu.md]: I would remove "Read world" to make this title simpler...

File: automobile/scenario-creation-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+## Creation of a real world scenario
```
Comment (omichel) [inline automobile/scenario-creation-tutorial.md]: I would keep the same title as the page name, e.g., "Scenario creation tutorial".

File: automobile/scenario-creation-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+This small tutorial will explain step by step how to create a real scenario inspired from a real world map and then add traffic using SUMO.
```
Comment (omichel) [inline automobile/scenario-creation-tutorial.md]: will explain &rarr; explains
step by step &rarr; step-by-step
I would remove "real" in "real scenario".

File: automobile/tools.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Tools
 
-This section will present various tools to be used for automobile simulation sin
-Webots.
+This section will present various tools to be used for automobile simulations in Webots and a tutorial explaining step by step how to create a real scenario inspired from a real world map.
```
Comment (omichel) [inline automobile/tools.md]: We should avoid using the future tense when not needed.
will present &rarr; presents
real scenario &rarr; scenario
step by step &rarr; step-by-step

File: automobile/tools.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
...
 
 ## Sections
 - [OpenStreetMap importer](openstreetmap-importer.md)
 - [SUMO interface](sumo-interface.md)
+- [Real world scenario creation tutorial](scenario-creation-tutorial.md)
```
Comment (omichel) [inline automobile/tools.md]: Fix the title to match the file name.

**Comment by DavidMansolino** [comment]
:+1:

File: automobile/scenario-creation-tutorial.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+%figure "Export a part of map from OpenStreetMap"
+![osm_export.png](images/osm_export.png)
+%end
+
+### Generate the Webots world
+
+As explained in the [OpenStreetMap importer](openstreetmap-importer.md) section, you should use the previously downloaded map to generate the Webots world.
+
+> **note**:
+This importer use splines to improve and smooth the path of the roads, unfortunately the OpenStreetMap to SUMO importer does not support this, it is therefore recommended to disable it (setting the spline subdivision to 0) if you want then to add traffic using SUMO.
```
Comment (stefaniapedrazzi) [inline automobile/scenario-creation-tutorial.md]: Typo: use -> uses

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**